api/PushManager.json

```diff
@@ -552,6 +552,66 @@
             "deprecated": true
           }
         }
+      },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {
+            "chrome": {
+              "version_added": "42"
+            },
+            "chrome_android": {
+              "version_added": "42"
+            },
+            "edge": [
+              {
+                "version_added": "16",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Enable service workers"
+                  }
+                ]
+              },
+              {
+                "version_added": "17"
```

Comment:
The more general support statement needs to go first (first version_added: 17, then the one behind a flag). I realize this was inherited from above, but I'd rather not duplicate the problem here.

Reply:
Fixed. So is the general rule that the most recent version in a set appears first? If so can the tooling notify of that?

Reply:
That's the general rule, but there's a bunch of subtleties that make automation a challenge. The real ranking is rather more complex. It's something like: most-recent available unflagged and unprefixed, then available prefixed, then available flagged, then anything removed, with ties going to things without contemporaneously applicable notes or partial flags. 😅 

It's probably better to think of the order as an authorial decision. What's most widely applicable and useful to web developers right now? That's probably what should go first.